src/forwarders.js

```diff
@@ -590,6 +605,7 @@ export default function Forwarders(mpInstance, kitBlocker) {
             );
         } else {
             try {
+                // Process MP configured Kits
```

Comment:
Consider splitting this function up into separate functions, such as configure, register and configure pixel, or at the very least break up the flow so that each "unit" is in its own try/catch block with unique error cases. I think this would make it easier to debug and possibly refactor later.

Reply:
good call